src/query/api/v1/handler/prom/read.go

```diff
@@ -39,15 +39,20 @@ import (
 	"go.uber.org/zap"
 )
 
+// RequestParser is a function that parses request arguments
```

Comment:
`.` at the end

Reply:
https://github.com/m3db/m3/pull/2884/commits/c70d0f506b1cc6b053e6fd086c5f70dc4aff3fe9#diff-cbccca18d3e7a3ac7e3867b53f6ed3673949a973ba1ddf3c882f721f81c842aaR42